ONSdigital/eq-survey-runner
# eq-363 Remove character limit from textarea

### What is the context of this PR?

Fixes #363.

This PR removes the character limit from comment (ie. textarea) questions.
### How to review
1. `npm run compile` to compile static assets.
2. visit a survey with comments (eg. `1_0205.json`)
3. confirm character limit UI has gone
4. submit more than 2000 characters
5. confirm no errors
### Who worked on the PR

@hamishtaplin
